java/integTest/src/test/java/glide/CommandTests.java

```diff
@@ -0,0 +1,24 @@
+package glide;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import java.util.concurrent.TimeUnit;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+
+public class CommandTests {
+
+    @Test
+    @SneakyThrows
+    public void custom_command_info() {
+        var regularClient =
+                RedisClient.CreateClient(
```

Comment:
we could probably move the CreateClient to a @BeforeAll or something - create the client once, and execute all the commands. 
Close after all is said and done?

Reply:
Fixed in 8896eb89 thatnks